addon/components/rdfa/toolbar-dropdown.js

```diff
@@ -1,23 +1,41 @@
 import Component from "@glimmer/component";
-import { action } from "@ember/object";
-import { guidFor } from '@ember/object/internals';
-import { tracked } from "@glimmer/tracking";
+import {action} from "@ember/object";
+import {guidFor} from '@ember/object/internals';
+import {tracked} from "@glimmer/tracking";
 
 export default class AuDropdown extends Component {
   // Create a dropdown ID
   dropdownId = 'dropdown-' + guidFor(this);
 
   // Track dropdown state
   @tracked dropdownOpen = false;
+  @tracked focusTrapActive = false;
+
+
+  get rootNode() {
+    return this.args.editor.rootNode;
+  }
+
+  activateFocusTrap() {
+    this.focusTrapActive = true;
+  }
+
+  @action
+  deactivateFocusTrap() {
+    this.focusTrapActive = false;
+  }
 
   @action
   deactivateDropdown() {
-    if (this.dropdownOpen)
```

Comment:
this was here for a reason (to avoid changing it twice in a runloop), is that solved if not reinstate it please :). things like these are typically there for a reason

Reply:
it actually errorred when I left it in :)
Note the way the dropdown is closed is different now: only the the deactivation of the focusTrap triggers this method
before there was an onclick on the div which deactivated the dropdown AND also deactivated the focusTrap, which again deactivated the dropdown
you can see how that can cause issues

Reply:
in that case I said nothing!